test_conformance/extensions/cl_khr_command_buffer/cl_khr_command_buffer_mutable_dispatch/mutable_command_simultaneous.cpp

```diff
@@ -329,7 +330,7 @@ struct SimultaneousMutableDispatchTest : public BasicMutableCommandBufferTest
         error = clEnqueueReadBuffer(work_queue, out_mem, CL_FALSE,
                                     pd.offset * sizeof(cl_int), data_size(),
                                     pd.output_buffer.data(), 1,
```

Comment:
We don't verify the contents of ` pd.output_buffer` from this call, before the second `clEnqueueReadBuffer` called that overwrites it, so this `clEnqueueReadBuffer` call doesn't add much testing value right now. Would it be worth adding a new std::vector member to `SimulPassData` so that both the `clEnqueueReadBuffer` outputs are stored and can verified afterwards? Otherwise, we may as well fix this race condition by removing this `clEnqueueReadBuffer` call.

Reply:
Thanks! I've separated the destination vectors for these two calls.